doc/releasenotes/16_0_0_summary.md

```diff
@@ -3,27 +3,27 @@
 ### Table of Contents
 
 - **[Major Changes](#major-changes)**
-  - **[VReplication](#vreplication)**
-    - [VStream Copy Resume](#vstream-copy-resume)
-    - [VDiff2 GA](#vdiff2-ga)
-  - **[Tablet throttler](#tablet-throttler)**
-  - **[Incremental backup and point in time recovery](#inc-backup)**
-  - **[Replication manager removal and VTOrc becomes mandatory](#repl-manager-removal)**
   - **[Breaking Changes](#breaking-changes)**
     - [VTGate Advertised MySQL Version](#advertised-mysql-version)
     - [Default MySQL version on Docker](#default-mysql-version)
     - [Running Vitess on the Operator](#running-vitess-on-the-operator)
```

Comment:
```suggestion
    - [Running Vitess with the Operator](#running-vitess-on-the-operator)
```

Reply:
Now that I've read this section, we should make this stronger
Something like `Upgrading to this release with vitess-operator`
or add ⚠️ at the beginning and end!!